Wikipedia/Code/WMFArticleListTableViewController.m

```diff
@@ -284,6 +284,10 @@ - (NSString *)saveActionText {
     return WMFLocalizedStringWithDefaultValue(@"article-save", nil, nil, @"Save", @"Text of the article list row action shown on swipe which allows the user to save the article");
```

Comment:
Added a ticket to track this issue: https://phabricator.wikimedia.org/T170938 - doesn't necessarily need to happen with this PR.